map/local_ads_manager.cpp

```diff
@@ -274,6 +275,51 @@ void LocalAdsManager::UpdateViewport(ScreenBase const & screen)
   }
 }
 
+bool LocalAdsManager::DownloadCampaign(MwmSet::MwmId const & mwmId, std::vector<uint8_t> & bytes)
+{
+  bytes.clear();
+
+  std::string const url = MakeCampaignDownloadingURL(mwmId);
+  if (url.empty())
+    return true;
```

Comment:
Am I right that previous version returned false? Please, add a comment to the method, it's not clear for me now when it return true and when false.